pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -668,6 +668,10 @@ type ThanosSpec struct {
 	BlockDuration Duration `json:"blockSize,omitempty"`
 	// ReadyTimeout is the maximum time Thanos sidecar will wait for Prometheus to start. Eg 10m
 	ReadyTimeout Duration `json:"readyTimeout,omitempty"`
+	// GetConfigInterval is how often the Prometheus config is fetched. Default 30s.
```

Comment:
I wouldn't mention the default value as it's defined by Thanos and may change.

```suggestion
	// How often to retrieve the Prometheus configuration.
```